R/geom-.R

```diff
@@ -63,6 +63,8 @@ Geom <- ggproto("Geom",
 
   default_aes = aes(),
 
+  default_params = NULL,
+
```

Comment:
1.) The Geom and Stat classes have a new field that can hold a named list of default parameters.